src/tests/ftest/util/exception_utils.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/python
 """
-  (C) Copyright 2020-2022 Intel Corporation.
+  (C) Copyright 2022 Intel Corporation.
```

Comment:
This date change appears incorrect.
```suggestion
  (C) Copyright 2020-2022 Intel Corporation.
```

Reply:
It's not actually.  It was incorrect before but this corrects it.  This file was created in 2022.